homeassistant/components/mazda/__init__.py

```diff
@@ -143,14 +142,10 @@ async def async_update_data():
         try:
             vehicles = await with_timeout(mazda_client.get_vehicles())
 
-            vehicle_status_tasks = [
-                with_timeout(mazda_client.get_vehicle_status(vehicle["id"]))
-                for vehicle in vehicles
-            ]
-            statuses = await gather_with_concurrency(5, *vehicle_status_tasks)
-
-            for vehicle, status in zip(vehicles, statuses):
-                vehicle["status"] = status
+            for vehicle in vehicles:
```

Comment:
Please make a code comment why we do this in sequence so we remember in the future.

Reply:
Comment added.